### ISSUE TITLE
Icon/Image support

### ISSUE THREAD

**Issue by FireCubeStudios** [FireCubeStudios/Protecc#5]
Ideally should also suggest icons based on the account type like how Pazzword does it. This is low priority for now

**Comment by DmitryGolubenkov** [comment]
I think for this functionality the app should also save issuer, when it is present in QR etc.
This would allow to automatically select one of integrated icons in future.

So - partly blocked by key otp issue.

A feature to manually add icons can still be done (and is required for auto icons)

**Comment by FireCubeStudios** [comment]
This issue will be hard to implement due to credential locker limitation
